### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by manish-GH**
Added FILTERS for the product listing page
[PREVIEW](https://prismatic-cheesecake-6bf36f.netlify.app/)

Things to look for
- DataContext.js
- ProductList folder

PS: TOASTS will be added later on

### CODE REVIEW

File: src/components/Slider/index.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+					disabled
+					value={price.minVal}
+					/* MIGHT REQUIRE THESE COMMENTED CODES LATER */
+					// onChange={(e) =>
+					// 	handleChange(
+					// 		e.target.value < min ? min : e.target.value,
+					// 		setMinVal,
+					// 		"min"
+					// 	)
+					// }
```
Comment (pradipgharge) [inline src/components/Slider/index.js:69]: Remove commented code

**Review by pradipgharge — APPROVED** [review]

File: src/components/Slider/index.js (hunk 1 above)
Comment (manish-GH) [reply src/components/Slider/index.js:69]: I may require this code in the future

**Review by Naman-Saxena1 — APPROVED** [review]
Overall code looks good and functionality is also working fine 👍 
1. You can make "Shop Now" stand out, right now it just merges with normal text.
2. Later if you get time, you can add steps to price slider, steps of Rs50 or Rs100

Capture1

Capture2

**Comment by manish-GH** [comment]
> Overall code looks good and functionality is also working fine 👍
> 
> 1. You can make "Shop Now" stand out, right now it just merges with normal text.
> 2. Later if you get time, you can add steps to price slider, steps of Rs50 or Rs100
> 
> Capture1
> 
> Capture2

For now I'll merge it, but I'll make a note of this for future tasks

**Pull request merged**